models/models.go

```diff
@@ -1,44 +1,22 @@
 package models
 
-import (
-	"fmt"
-	"gorm.io/driver/sqlite"
-	"gorm.io/gorm"
-	"log/slog"
-)
-
-var DbName = "db.sqlite3"
+import "gorm.io/gorm"
 
 type User struct {
 	gorm.Model
-	ExternalID uint
+	ExternalID int64 `gorm:"not null"`
 }
 
 type Repository struct {
 	gorm.Model
-	Url       string
-	LatestTag string
+	Url       string `gorm:"size:100;not null;unique"`
```

Comment:
`Owner/Name` чтобы не парсить раз за разом URL. Mожет вообще не хранить URL, а строить его из отдельных кусков? 
```suggestion

	URL       string `gorm:"size:100;not null;unique"`
	Owner string
	Name string
```